lib/apipie/application.rb

```diff
@@ -55,8 +55,7 @@ def rails_routes(route_set = nil)
     # this method does in depth search for the route controller
     def route_app_controller(app, route, visited_apps = [])
       if route.defaults[:controller]
-        controller_name = (route.defaults[:controller] + 'Controller').camelize
-        controller_name.safe_constantize
+        (route.defaults[:controller].camelize + 'Controller').safe_constantize
```

Comment:
Just checked what rails actually does with mixed case stuff when camelizing. Definetly makes more sense this way.